README.md

```diff
@@ -58,5 +42,120 @@ You can also track upcoming changes using the [awaiting-deployment](https://gith
 [macOS-10.15]: https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md
 [self-hosted runners]: https://help.github.com/en/actions/hosting-your-own-runners
 
-## Software and image guidelines
-To learn more about tools and images support policy, see the [guidelines](./docs/software-and-images-guidelines.md).
+## Image Definitions
+### Beta:
+The purpose of a Beta is to collect feedback on an image before it is released to GA. The goal of a Beta is to identify and fix any potential issues that exist on that
+image. Images are updated on a weekly cadence. Any workflows that run on a beta image do not fall under the customer [SLA](https://github.com/customer-terms/github-online-services-sla) in place for Actions.
+Customers choosing to use Beta images are encouraged to provide feedback in the runner-images repo by creating an issue. A Beta may take on different availability, i.e. public vs private.
+
+### GA:
+A GA (General Availability) image has been through a Beta period and is deemed ready for general use. Images are updated on a weekly cadence. In order to be moved to
+GA the image must meet the following criteria: 1- Has been through a Beta period (public or private) 2- Most major software we install on the image has a compatible
+version for the underlying OS and 3- All major bugs reported during the Beta period have been addressed. This image type falls under the customer [SLA](https://github.com/customer-terms/github-online-services-sla) for actions. GA images are eventually deprecated according to our guidelines as we only support the
+latest 2 versions of an OS.

```

Comment:
Should we add data about Deprecated images here as well?

Reply:
I'm not sure that's necessary as we cover deprecated images elsewhere under the Image Deprecation Policy

Reply:
I'd recommend having all image definitions in one section. Haven't gotten to the deprecation policy yet, but will take a look and change my comment as needed.

Reply:
Going to leave as-is for now, we can revisit the idea if we think it is worthwhile to add.